AirCasting/SDSync/MobileSessions/SDCardMobileSessionsSavingService.swift

```diff
@@ -189,9 +193,8 @@ class SDCardMobileSessionsSavingService: SDCardMobileSessionssSaver {
     }
     
     private func enqueueForSaving(measurements: SDCardMeasurementsRow, buffer streamsWithMeasurements: inout [SDStream: [SDSyncMeasurement]], deviceID: String) {
-        var location: CLLocationCoordinate2D?
-        if let lat = measurements.lat, let long = measurements.long {
-            location = CLLocationCoordinate2D(latitude: lat, longitude: long)
+        if location == nil {
```

Comment:
maybe guard let location else { ... } instead if this check?

Reply:
Hmm, I think no. We do not return anything inside this and guard requires return/throw. 
Or maybe there's something I don't see here?

Reply:
True dat, my bad. 

I'm used to syntax like that: 
` location == location ?? setProperLocation(deviceID: deviceID, measurements: measurements)`

but your call, I do't mind either way.

Reply:
Changed 🙌🏻